Patches/OpenCV/3.1.0/common.cmake

```diff
@@ -0,0 +1,150 @@
+# This file is included from a subdirectory
```

Comment:
I added a commit that reverts to the original version and then another one that applies the patch. Hopefully that works. (Isn't all of this going to get squashed on merge anyway)?